lib/edoc/src/edoc.erl

```diff
@@ -55,26 +55,31 @@
 	 read_comments/1, read_comments/2,
 	 read_source/1, read_source/2]).
 
--export_type([module/0]).
+-export_type([edoc_module/0]).
 
 -compile({no_auto_import,[error/1]}).
 
 -include("edoc.hrl").
+-include_lib("xmerl/include/xmerl.hrl").
 
 
--opaque edoc_module() :: term().
+%% The EDoc documentation data for a module,
```

Comment:
The comment needs to be after the <code>-opaque</code> definition to generated into the documentation (edoc still uses edoc for documentation :) )